ui/package.json

```diff
@@ -29,7 +29,7 @@
     "react-dom": "16.4.2"
   },
   "devDependencies": {
-    "@connexta/ace": "git://github.com/connexta/ace.git#eaebb0fcbd5d4d9f97ced0ff9c3611ce1497fab6",
+    "@connexta/ace": "git://github.com/connexta/ace.git#8d4bb2ed68a4c4fe2c20b418914fd5fe1f3204c9",
```

Comment:
❓ Is the long term plan to continue to work with commit hashes for ace, or are we looking at supporting released versions?

Reply:
I wasn't planning on releasing a specific version of ace because we keep making incremental improvements and it only adds an extra step for bringing those changes into DDF. We have been using hash versions for a while now and it hasn't caused any issues so I don't see a reason for change. However, DDF master should only depend on master commits of ace otherwise the commit isn't guaranteed to be in the repo.